protocol/daemons/liquidation/client/sub_task_runner.go

```diff
@@ -203,6 +207,49 @@ func (c *Client) GetLiquidatableSubaccountIds(
 		if hasNegativeTnc {
 			negativeTncSubaccountIds = append(negativeTncSubaccountIds, *subaccount.Id)
 		}
+	}
+
+	return liquidatableSubaccountIds, negativeTncSubaccountIds, nil
+}
+
+// GetSubaccountOpenPositionInfo iterates over the given subaccounts and returns a map of
+// perpetual id to open position info.
+func (c *Client) GetSubaccountOpenPositionInfo(
+	subaccounts []satypes.Subaccount,
+) (
+	subaccountOpenPositionInfo map[uint32]clobtypes.SubaccountOpenPositionInfo,
```

Comment:
Sorry I might have missed this in a different PR, but why did we choose to go with a value instead of a pointer for the values in the subaccountOpenPositionInfo map?

Reply:
yup makes sense - also good for consistency since we are already using pointers on the server side. updated!